src/main/cliapp/src/components/Editors/NotEditor.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Dropdown } from "primereact/dropdown";
+
+export function NotEditor({ props, value, editorChange }) {
+	const [selectedValue, setSelectedValue] = useState(value);
+	const textString = value ? "NOT" : "";
+	const options = [
+		{ label: "NOT", value: true },
+		{ label: "null", value: false }
```

Comment:
So the idea is in the pull down you will either select "NOT" or you will not select anything at all. In other words there is 1 option in the pull down its either selected or its not.

Reply:
Okay, sure